src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2427,6 +2548,43 @@ def aks_rotate_certs(cmd, client, resource_group_name, name, no_wait=True):
     return sdk_no_wait(no_wait, client.rotate_cluster_certificates, resource_group_name, name)
 
 
+def fuzzy_match(query, arr):
+    def similarWord(a, b):
+        aLen = len(a)
+        bLen = len(b)
+        if aLen > bLen:
+            return similarWord(b, a)
+        if a == b:
```

Comment:
We can put `a==b` and `a in b` check before length check. And what if `b in a` ? It's highly recommended If we can give some examples and comments here to help others understand the logic.

Reply:
If `bLen > aLen`, then there is no chance of `b in a` where `b != a`.

Reply:
But I could delete the `a == b`. I just left it in for clarity.